junit-jupiter-api/src/main/java/org/junit/jupiter/api/DisplayNameGenerator.java

```diff
@@ -88,6 +107,11 @@ static String parameterTypesAsString(Method method) {
 	 */
 	class Standard implements DisplayNameGenerator {
 
+		/**
+		 * Pre-defined standard display name generator instance.
+		 */
+		static final DisplayNameGenerator standardGenerator = new Standard();
```

Comment:
Let's just call it `INSTANCE` (same below).

Reply:
Roger, lees repetitive when using it.